net/seafile-server/Makefile

```diff
@@ -24,6 +24,9 @@ include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/nls.mk
 include ../../lang/python/python-package.mk
 
+SEAHUB_CUR_VERSION:=$(shell egrep '^PKG_VERSION[ :]*=' ../seafile-seahub/Makefile | sed 's/PKG_VERSION.*=[ ]*//')
+SEAHUB_CUR_RELEASE:=$(shell egrep '^PKG_RELEASE[ :]*=' ../seafile-seahub/Makefile | sed 's/PKG_RELEASE.*=[ ]*//')
+
```

Comment:
Maybe we should just `$(error xxx)` if seafile-seahub version does not match that of seafile-server

Reply:
I will leave the matching logic, and error out if they don't match.  Compile-time errors are a lot easier to catch than install-time errors.
Thanks for the review.